service_report/Identify_updated_endpoints.ipynb

```diff
@@ -0,0 +1,276 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d918690d-07cf-4ee6-abdc-a6f85b4faa71",
+   "metadata": {},
+   "source": [
+    "This Report gets a list of endpoints that have updated along with the entity count."
```

Comment:
I think there was talk of a template description for Jupyter Notebooks, but I'm not sure!

Reply:
Oh yes
Thanks for reminding!